dev/plugins/hu.elte.txtuml.api.stdlib/src/hu/elte/txtuml/api/stdlib/timers/Timer.xtxtuml

```diff
@@ -28,18 +35,38 @@
 	 *            the signal which is to be sent after the delay
 	 * @param millisecs
 	 *            the time in millisecs to wait before sending the signal
-	 * @return a timer instance to manage this delayed send operation before it
+	 * @return a handle object to manage this delayed send operation before it
 	 *         happens
 	 */
-	static Timer start(ModelClass targetObj, Signal signal, int millisecs) {
-		return new TimerImpl(targetObj, signal, millisecs);
+	public static external-body Timer start(Signal sig, ModelClass targetObj, int millisecs) {
```

Comment:
The `create` here is the `Action.create` API method which in the exported UML2 model is an object creation action followed by a constructor call. However, the constructor which is called here is external.

Note that the methods of the `Action` class are all treated specially when the model is exported to UML2. They are marked to have an external body only to emphasize for the users that they can be used in the model but their implementation is not modeled. It may also simplify the validation of action language elements.

Another reason to use `external-body` here and also in the case of the `add` method is that the `Timer` class has been intended to be just an interface in the model above an external feature. When exporting a model, the target language has complete freedom in how to implement this class.